include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -120,24 +121,61 @@ class IGPUObjectFromAssetConverter
             // @sadiuk put here more parameters if needed
 
             SPerQueue perQueue[EQU_COUNT];
+            
+            inline void beginCommandBuffer(E_QUEUE_USAGE queueUsage)
+            {
+                uint32_t queue = static_cast<uint32_t>(queueUsage);
+                assert(perQueue[queue].cmdbuf);
+                perQueue[queue].cmdbuf->begin(IGPUCommandBuffer::EU_ONE_TIME_SUBMIT_BIT);
+            }
 
-
+            inline void beginCommandBuffers()
+            {
+                for (auto i=0; i<EQU_COUNT; i++)
+                {
+                    beginCommandBuffer(static_cast<E_QUEUE_USAGE>(i));
+                }
+            }
+            
+            inline void endUnusedCommandBuffers()
+            {
+                for(auto & queue : perQueue)
+                {
+                    if(queue.cmdbuf->getState() == IGPUCommandBuffer::ES_RECORDING)
```

Comment:
guard with an `if` because its perfectly fine to optionally give cmdbuffers (like I dont want to have to specify all 4 commandbuffers if I only want to make a GPU Pipeline from a CPU pipeline)